jerry-core/CMakeLists.txt

```diff
@@ -180,6 +180,7 @@ project (JerryCore CXX C ASM)
    target_include_directories(${TARGET_NAME}.jerry-core PRIVATE ${INCLUDE_CORE})
    target_include_directories(${TARGET_NAME}.jerry-core PRIVATE ${INCLUDE_FDLIBM})
    target_include_directories(${TARGET_NAME}.jerry-core SYSTEM PRIVATE ${INCLUDE_LIBC_INTERFACE})
+   target_include_directories(${TARGET_NAME}.jerry-core SYSTEM PRIVATE ${INCLUDE_EXTERNAL_LIBS_INTERFACE})
```

Comment:
For the RTOS I'm working, there are multiple include paths needed to compile besides `include` with stdlib.h.